### ISSUE TITLE
Web tutorial not working

### ISSUE THREAD

**Issue by kylooh** [JetBrains/compose-jb#1642]
I'm follow [Web/Getting_Started](https://github.com/JetBrains/compose-jb/tree/master/tutorials/Web/Getting_Started) tutorial's instructions, and received errors while running `./gradlew jsBrowserRun` 

```
* What went wrong:
Could not determine the dependencies of task ':kotlinNodeJsSetup'.
> Could not resolve all files for configuration ':detachedConfiguration1'.
   > Could not find org.nodejs:node:14.17.0.
     Searched in the following locations:
```

My computer informations:
> hardware: MacBook Pro (13-inch, M1, 2020)
> OS: macOS Big Sur 11.3.1
> IDE: IntelliJ IDEA 2021.2
> JDK: Azul Zulu Community 11.0.12
> Kotlin Plugin Version: 212-1.5.10-release-IJ4746.92

**Comment by akurasov** [comment]
see https://github.com/JetBrains/compose-jb/issues/1625 

Closing as duplicate

**Comment by jollygreenegiant** [comment]
TL;DR add this to your gradle file, plugging in whatever the current version of Node is

```
rootProject.plugins.withType<org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootPlugin> {
    rootProject.the<org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootExtension>().nodeVersion = "16.17.1"
}
```

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-1642">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
